### PR TITLE
upgrade to lighthouse v6.4.1

### PR DESCRIPTION
**Pull request by iandeveseleer**
Upgrade to latest Lighthouse version.

In particular to fix an issue with accessibility, caused by the lighthouse's axe-core dependency: https://github.com/dequelabs/axe-core/issues/2483

### CODE REVIEW

**Comment by mikestead** [comment]
Hey thanks @iandeveseleer , if it's a breaking change we'd need a major version bump though. I assume it wasn't breaking if lighthouse didn't do a major bump?

File: package.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-    "type": "git",
-    "url": "https://github.com/mikestead/lighthouse-batch.git"
-  },
-  "dependencies": {
-    "commander": "^2.9.0",
-    "lighthouse": "^6.0.0",
-    "shelljs": "^0.8.4"
-  }
+    "name": "lighthouse-batch",
+    "version": "6.4.1",
```
Comment (mikestead) [inline package.json:3]: looks like you changed the whitespace in the file?

Also you boosted the lib version to align with lighthouse version, but this lib is independent in its versioning. We just happen to track the same major version as it's nice and easy to understand, and we inherit the breaking changes from lighthouse.

If you could leave the versioning of this lib and just update the dependency version of lighthouse. I can do the lib versioning when I release next.

Comment (iandeveseleer) [reply package.json:3]: Hey, it's not actually a major version of Lighthouse, it doesn't make any breaking change, I had just taken your text from the last lighthouse upgrade commit by wanting to remain coherent 😄.

I'm going to restore the version of the library as well as the whitespaces that have been modified at save with my IDE.

**Comment by mikestead** [comment]
perfect, thanks @iandeveseleer I'll get this released shortly

**Comment by mikestead** [comment]
released under v6.1.0

**Pull request merged**